docs/api/cloudlink/packets/ulist.md

```diff
@@ -0,0 +1,31 @@
+# `ulist`
+
+This packet provides a string of online users separated by ";"
+## Payload
+
+A string of users separated by ";"
+```
+user1;user2;user3;
+```
+
+## Usage
+::: code-group
```

Comment:
1. Is this section necessary at all? 
2. All the code blocks should remove the last element as well as the ulist puts a semicolon at the end

Reply:
1. Probably not, I can remove it.